cycledash/static/js/runs/components/RunsPage.js

```diff
@@ -308,13 +308,14 @@ var RunLabels = React.createClass({
                       .object()
                       .value();
     var labelTypes = [
-      ['tumor_bam_id', 'tumor', 'Has an associated tumor BAM'],
-      ['normal_bam_id', 'normal', 'Has an associated normal BAM'],
-      ['run', '', 'Has a running worker task'],
-      ['fail', '', 'Has a failed worker task']
+      [['tumor_bam' , 'name'], null, 'Tumor BAM'],
```

Comment:
The `null` vs `''` distinction is a little cryptic here. How about inlining the calls to be more direct? Something like

```javascript
var get = k => utils.getIn(run, k) || utils.getIn(taskStates, k),
    tumorName = get(['tumor_bam', 'name']),
    labelTypes = [
      [tumorName, tumorName, 'Tumor BAM'],
      [get(['run']), '', 'Has a running worker task'],
      ...
    ];
```